playbooks/azure/openshift-cluster/create_offer.yml

```diff
@@ -76,8 +76,20 @@
         msg: "{{ yeditout }}"
         verbosity: 1
 
-  - name: create an offer in cloudpartner portal
+    # this cancel operation returns a 202 whether it cancelled or not.
+    - name: cancel publish operation
```

Comment:
indent wrong?

Reply:
also I think I'd rename the playbook filename, it's doing much more than create at this point

Reply:
renamed.

Reply:
This is a continuation of the block above.  I believe this is correct.

Reply:
I believe it is correct as it is part of the previous block. The block states that if there is an offer already there then run these steps which includes a cancel operation. If the offer does not exist then there is no need to run the cancel operation.